sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2792,6 +2792,51 @@ object GpuOverrides {
           aggBuffer.copy(dataType = c.dataType)(aggBuffer.exprId, aggBuffer.qualifier)
         }
       }),
+    expr[ApproximatePercentile](
+      "Approximate percentile",
+      ExprChecks.fullAggAndProject(
```

Comment:
This should not work for project, and it is not supported for reduce yet, so we probably should add a new category to ExprChecks that only enables this for regular group by aggregations, not reductions, and not window operations, and definitely not project.